JS/Lesson20/gulcin/exercises.js

```diff
@@ -67,4 +67,76 @@ console.log(square);
 
 // 3. Create a Vehicle class with make, model, and year properties, and a start() method that logs a message to the console indicating that the vehicle has started. Then create a Car class that extends the Vehicle class and adds a numDoors property. Override the start() method in the Car class to log a different message to the console indicating that the car has started.
 
+class Vehicle {
+    constructor(make, model, year) {
+        this._make = make;
+        this._model = model;
+        this._year = year;
+    }
+    start() {
+        return `The vehicle has started!`;
+    }
+}
+
+class Car extends Vehicle {
+    constructor(make, model, year, numDoors) {
+        super(make, model, year)
+        this.numDoors = numDoors;
```

Comment:
Don't forget to mark internal values with "_", it makes code more maintanable :) 
```suggestion
        this._numDoors = numDoors;
```